javascriptv3/example_code/cross-services/transcription-app/src/index.ts

```diff
@@ -84,93 +90,91 @@ const s3Client = new S3Client({
+          new ListObjectsCommand({ Bucket: params.Bucket, Prefix: username })
+        );
+        // Create variable for the list of objects in the Amazon S3 bucket.
+        const output = data.Contents;
+        // Loop through the objects, populating a row on the user interface for each.
+        for (var i = 0; i < output.length; i++) {
+          var obj = output[i];
+          const objectParams = {
+            Bucket: params.Bucket,
+            Key: obj.Key,
+          };
+          // Get the name of the object from the Amazon S3 bucket.
+          const data = await s3Client.send(new GetObjectCommand(objectParams));
+          // Extract the body contents from the returned data. This is a readable stream.
+          const result = data.Body;
+          // Create variable for the string version of the readable stream.
+          let stringResult = "";
+          // Use 'yeidlUnit8Chunks' to convert readable streams into JSON.
+          for await (let chunk of yieldUint8Chunks(result)) {
+            stringResult += String.fromCharCode.apply(null, chunk);
+          }
+          // setTimeout function to wait while readable stream is converted into JSON.
+          setTimeout(function () {
+            // Parse JSON into human readable transcript, which will be displayed on user interface (UI).
+            const outputJSON = JSON.parse(stringResult).results.transcripts[0]
+              .transcript;
+            // Create name for transcript, which will be display on UI.
+            const outputJSONTime = JSON.parse(stringResult)
+              .jobName.split("/")[0]
+              .replace("-job", "");
+            i++;
+            //
+            // Display the details for the transcription on the UI.
+            // 'displayTranscriptionDetails()' is in 'helper.ts'.
+            displayTranscriptionDetails(
+              i,
+              outputJSONTime,
+              objectParams.Key,
+              outputJSON
+            );
+          }, 1000);
         }
-        // Wrap in setTimeout function to provide time for string to be parsed.
-        setTimeout(function () {
-          // Your code to be executed after 1 second
-          const outputJSON = JSON.parse(stringResult).results.transcripts[0]
-            .transcript;
-          const outputJSONTime = JSON.parse(stringResult)
-            .jobName.split("/")[0]
-            .replace("-job", "");
-          i++;
-          displayTranscriptionDetails(
-            i,
-            outputJSONTime,
-            objectParams.Key,
-            outputJSON
-          );
-        }, 1000);
+      } catch (err) {
+        console.log("Error", err);
       }
     } catch (err) {
-      console.log("Error", err);
+      console.log("Error creating presigned URL", err);
     }
   } catch (err) {
-    console.log("Error creating presigned URL", err);
+    console.log("Error", err);
   }
 };
 
-// Convert readable streams.
+// Convert readable streams
```

Comment:
This should probably says, "Convert the readable streams."
This is another instance where the period was removed.